app/routines/push_salesforce_lead.rb

```diff
@@ -33,7 +34,8 @@ def exec(user:, email: nil, role:, phone_number:, school:, num_students:, using_
       website: url,
       adoption_status: using_openstax,
       num_students: num_students.to_i,
-      os_accounts_id: user.id
+      os_accounts_id: user.id,
+      application_source: source_application.try(:lead_application_source) || ''
```

Comment:
Is a blank string what salesforce expects when we don't know the source?

Reply:
Also, does SF accept any string or just some specific strings?

Reply:
specific strings defined by SF admin (denver et al)

Reply:
blank string works when we don't know the string